libs/knowledge-store/ragstack_knowledge_store/graph_store.py

```diff
@@ -482,39 +476,33 @@ def fetch_initial_candidates() -> None:
                 # If the next nodes would not exceed the depth limit, find the
                 # adjacent nodes.
                 #
-                # TODO: For a big performance win, we should track which tags we've
+                # TODO: For a big performance win, we should track which links we've
```

Comment:
It seems like this may be a stale comment. Specifically, the `difference_update` on line 487 seems to be doing this.